tests/test_forward_one_to_one_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,164 @@
+    amz_author_profile = models.AmazonAuthorProfile.objects.create(
+        author=author,
+        description="Some description",
+    )
+    response = client.patch(
+        f"/api/amazon_author_profiles/{amz_author_profile.id}",
+        content_type="application/json",
+        data={"author": None},
+    )
+    assert response.status_code == status.HTTP_200_OK, response.json()
+    amz_author_profile.refresh_from_db()
+    assert amz_author_profile.author_id == None
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)